### ISSUE TITLE
Link-name fails with role="presentation" content

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#767]
The following snippet should not fail the link-name rule:

```html
<a href="link">
  <span role="presentation">Link text</span>
</a>
```
